wordpress-mobile/WordPress-iOS
# Fix layout of Quick Start table view header

Fixes the layout issues mentioned [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/11028#issuecomment-464654733)

<img width="341" alt="image" src="https://user-images.githubusercontent.com/517257/53070838-3d9d4800-3495-11e9-825e-93bef6a51a81.png">

<img width="415" alt="image" src="https://user-images.githubusercontent.com/517257/53070850-45f58300-3495-11e9-8a86-d51fe8c67818.png">


To test:
- View a site with Quick Start enabled, or add `return true` in `shouldShowQuickStartChecklist()`
- If you're Sylvester: Ensure the heading matches desired layout
- Otherwise: ensure the app still functions and the code is good

Update release notes:
